src/main/webapp/dataverse_template.xhtml

```diff
@@ -79,6 +79,7 @@
             </f:facet>
         </p:ajaxStatus>
         <ui:include src="contactFormFragment.xhtml"></ui:include>
+        <ui:include src="addDataFormFragment.xhtml"></ui:include>
```

Comment:
I was wondering what the best way to handle this popup is, and I think this a great solution. 👍